browser_patches/firefox/juggler/components/juggler.js

```diff
@@ -79,27 +79,37 @@ CommandLineHandler.prototype = {
       loadFrameScript();
       dump(`Juggler listening on ws://127.0.0.1:${this._server.port}/${token}\n`);
     } else if (jugglerPipeFlag) {
+      let browserHandler;
+      let pipeStopped = false;
       const pipe = Cc['@mozilla.org/juggler/remotedebuggingpipe;1'].getService(Ci.nsIRemoteDebuggingPipe);
       const connection = {
         QueryInterface: ChromeUtils.generateQI([Ci.nsIRemoteDebuggingPipeClient]),
         receiveMessage(message) {
           if (this.onmessage)
             this.onmessage({ data: message });
         },
+        disconnected() {
+          if (browserHandler)
```

Comment:
Can it actually be undefined in practice?

Reply:
Unlikely!